src/amber/controller/helpers/responders.cr

```diff
@@ -50,9 +50,12 @@ module Amber::Controller::Helpers
 
       private def select_type
         raise "You must define at least one response_type." if @available_responses.empty?
-        # NOTE: If only one response is requested don't return something else.
-        @requested_responses << @available_responses.keys.first if @requested_responses.size != 1
-        @requested_responses = @available_responses.keys if @requested_responses.includes?("*/*")
+        # NOTE: If */* is requested - make all available, but if only one response is requested don't return something else
+        if @requested_responses.includes?("*/*")
+          @requested_responses = @available_responses.keys
```

Comment:
The issue here is that if a request header is something like: `application/json;*/*;` then this will likely return `text/html` - which does not seem ideal.

I think we need to come to an agreement on the requirements for this and maybe even psuedo-code for how things should work.

Reply:
What about something like below? And if the requested response is not found in the available responses, shouldn't it be a 406?

```crystal
@requested_responses.find do |resp|
  @available_responses.keys.includes?(resp) || resp.includes?("*/*")
end
```